src/pages/FeedPage/FeedPage.jsx

```
@@ -1,34 +1,41 @@
-  const location = useLocation()
+  useEffect(() => {
+    if (currentPage !== navIndex.FEED) setCurrentPage(navIndex.FEED)
+    return () => setCurrentPage(navIndex.NONE)
+  }, [])
+
   useEffect(() => {
     const deleteDataState = (state) => {
-      if (!state || !state.usr) return state
+      if (!state?.usr) return state
       const newState = { ...state, usr: { ...state.usr } }
       if ('data' in newState.usr) delete newState.usr.data
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)